editor/js/Loader.js

```diff
@@ -766,6 +766,36 @@ function Loader( editor ) {
 
 			}
 
+			case 'png':
+			case 'jpg':
+			case 'jpeg':
+			case 'bmp':
+			case 'webp':
+			{
+
+				const image = new Image();
+				image.src = URL.createObjectURL( file );
+
+				image.onload = function () {
+
+					const texture = new THREE.Texture( image );
+					texture.colorSpace = THREE.SRGBColorSpace;
```

Comment:
Um, what if the user imports a normal map?

Reply:
> Um, what if the user imports a normal map?

Maybe this PR can make it like blender: `File -> Import -> Images as Planes`. So there's no confusion.

ScreenShot:
![image](https://github.com/mrdoob/three.js/assets/23699926/c6b1cb90-0e70-4767-864c-fdc4a8e568f9)

Reply:
Sounds good!